eslint-plugin/index.ts

```diff
@@ -31,6 +25,6 @@ const voicevoxPlugin: VoicevoxPlugin = {
       },
     ],
   },
-};
+} as const satisfies FlatConfig.Plugin;
```

Comment:
試してみた感じ、as constは別に無くてもいいみたいですね。
何か別の理由で付けているのであれば、すみません。
```suggestion
} satisfies FlatConfig.Plugin;
```

Reply:
なくても動くならなくても良さそう。